src/UI/FSMPMenu.cpp

```
@@ -0,0 +1,1582 @@
+
+		// A live view of the FSMP log's tail, so the log (including [SMP Metrics] and profiler results) can be
+		// read without leaving the game. Only the last chunk is loaded, so a report-bloated log stays cheap.
+		section(fa::FileLines, "Log file");
+		static std::string logText;
+		static bool autoRefresh = true;
+		static int frame = 0;
+		bool doRead = ImGuiMCP::Button(tr("Refresh"));
+		ImGuiMCP::SameLine();
+		ImGuiMCP::Checkbox(tr("Auto-refresh"), &autoRefresh);
+		ImGuiMCP::SameLine();
+		ImGuiMCP::TextDisabled(tr("(last %d KB)"), 256);
```

Clang (14.0.6)

[warning] do not call c-style vararg functions

(cppcoreguidelines-pro-type-vararg)